src-java/base-topology/base-messaging/src/main/java/org/openkilda/messaging/payload/flow/FlowEncapsulationType.java

```diff
@@ -15,11 +15,68 @@
 
 package org.openkilda.messaging.payload.flow;
 
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
 /**
  * Class represents flow encapsulation type.
  */
 public enum FlowEncapsulationType {
-    TRANSIT_VLAN,
-    VXLAN
+    /**
```

Comment:
I don't think it's a good idea to modify app code for tests. May be it's better to copy this enum to testing.* package?